message TimeoutMsg {
  required string wakeup = 1;
}